compiler/testData/diagnostics/tests/annotations/options/targets/field.fir.kt

```diff
@@ -28,7 +28,7 @@ abstract class My(@Field arg: Int, @Field val w: Int) {
     fun foo() {}
 
     @Field
-    val v: Int by <!UNRESOLVED_REFERENCE!>Delegates<!>.lazy { 42 }
+    val v: Int by <!DELEGATE_SPECIAL_FUNCTION_MISSING!><!UNRESOLVED_REFERENCE!>Delegates<!>.lazy { 42 }<!>
```

Comment:
It's still `UNRESOLVED_REFERENCE` in FE 1.0: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/annotations/options/targets/field.kt#L31

looks like we need some way not to report `...MISSING` for unresolved reference?